app/views/plans/show_export.html.erb

```diff
@@ -11,10 +11,10 @@
     <!--Text describing page content-->
     <%= raw _("<p>From here you can  download your plan in various formats. This may be useful if you need to submit your plan as part of a grant application.</br> Select what format you wish to use and click to 'Export'.</p>")%>
 
-    <% if @plan.phases.count == 1 %>
-        <%= render :partial => "/shared/export_links", locals: {plan: @plan, phase: @phan.phases[0]} %>
+    <% if @plan.template.phases.count == 1 %>
```

Comment:
This is a valid comment but since it isn't actually a bug I will accept the PR for now and then we can discuss how we want to handle things like this.